snp_motion_planning/src/plugins/tasks/tcp_speed_limiter_profile.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+static const std::string TCP_SPEED_LIMITER_TASK_NAME = "TCPSpeedLimiterTask";
+
+namespace snp_motion_planning
+{
+struct TCPSpeedLimiterProfile
+{
+  using Ptr = std::shared_ptr<TCPSpeedLimiterProfile>;
+  using ConstPtr = std::shared_ptr<const TCPSpeedLimiterProfile>;
+
```

Comment:
Let's add a default constructor as well.

```suggestion
TCPSpeedLimiterProfile() = default;
```